src/Section2/style2.js

```diff
@@ -34,31 +55,38 @@ padding-bottom: 60px
 background: rgba(255, 214, 108, 0.12);
 box-shadow: 1px 1px 30px rgba(0, 0, 0, 0.25);
 border-radius: 10px;
-`
 
-export const Html5Icon = styled(HTML5)`
+@media (max-width: 768px;){
+  width: 191.27px;
+  height: 137px;
+}
 
-width: 92px;
-height: 92px;
-margin-right: 61px;
-margin-left:62px;
-margin-top: 31px;
-`
+@media (max-width: 376px;){
+  width: 191.27px;
+  height: 137px;
+}
+`;
 
-export const CSS3Icon = styled(CSS3)`
+export const Html5Icon = styled(HTML5)`
+  width: 92px;
+  height: 92px;
+  margin-right: 61px;
+  margin-left: 62px;
+  margin-top: 31px;
+`;
 
-width: 92px;
-height: 92px;
-margin-right: 61px;
-margin-left:62px;
-margin-top: 31px;
-`
+export const CSS3Icon = styled(CSS3)`
+  width: 92px;
+  height: 92px;
+  margin-right: 61px;
+  margin-left: 62px;
+  margin-top: 31px;
+`;
 
 export const JSIcon = styled(JS)`
-width: 92px;
-height: 92px;
-margin-right: 61px;
-margin-left:62px;
-margin-top: 31px;
-
-`;
\ No newline at end of file
+  width: 92px;
+  height: 92px;
+  margin-right: 61px;
+  margin-left: 62px;
+  margin-top: 31px;
+`;
```

Comment:
All the icons have the exact same code. You could just reuse this.

If you know the icon inside of the box will always have the same CSS you can change the CSS of the box to something like:
```
const Box1 = styled.div`
  ...
  & > svg {
    width: 92px;
  height: 92px;
  margin-right: 61px;
  margin-left: 62px;
  margin-top: 31px;
  }
`
This way all the svgs inside of the box will take this CSS. This is like a selector. The `&` means whatever is the selector I am inside + whatever I am defining now.

So & > svg in side case means `div > svg`
```